packages/worker-activity-page/src/controllers/activity.tsx

```diff
@@ -0,0 +1,40 @@
+import {ActivityPage} from "../components/activity-page" 
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/alexwilson/frontend/pullRequest?prid=7165000)
```suggestion
import {ActivityPage} from "../components/activity-page"; 
```